canonical-web-and-design/maas-ui
# feat(ui) machine details commissioning table

## Done

- Build the machine details commissioning table

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go into a machine
- Enter the commissioning table
- See that it matches the same machine in bolla
- Expand the result and see they are there without the repeated content of name and node.

## Fixes
Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/2212
